infra/docker/web/Dockerfile

```diff
@@ -0,0 +1,101 @@
+FROM php:7.4-fpm-alpine
+
+EXPOSE 80
+
+# Create some environment variables if not present.
+# TBD - We're going to be injecting a lot of values via GitHub Actions.
+
+# Configure timezone
+RUN ln -s /usr/share/zoneinfo/Etc/UTC /etc/localtime
+
+# Install Libraries
+RUN apk add --update --no-cache \
+    libgd \
+    imagemagick \
+    html2text \
+    nginx \
+    freetype \
+    postgresql \
+    tidyhtml
+
+# Install PHP Extensions not already present in alpine release
+# RUN docker-php-ext-install -j$(nproc) \
+#     opcache \
+#     pgsql \
+#     tidy \
+#     gd
```

Comment:
Why is this commented out? Is this going to be enabled in the next PR?

I want to avoid commented-out legacy code, since later on it's unclear if it should be deleted or should be added.